### ISSUE TITLE
Crafting recipes using materials that create items should put them in your hands when possible instead of always dropping them on the floor.

### ISSUE THREAD

**Issue by PetMudstone** [unitystation/unitystation#6978]
### Description
Right now, when you craft items using materials such as metal sheets the item always gets dropped on the floor at your feet. This is inconvenient. 
Ideally when you craft an item, such as floor tiles using metal, they should try to fit in any empty hands you have available. If you completely use up a stack of materials when making an item and there are no other empty hands, then the crafted item should go in the hand that was holding the stack of materials. Items should only drop on the floor if there are no empty hands *after* you finish making the item in question.

**Comment by ThatDan123** [comment]
Crafting recipes which put items in hands should probably go into the crafting screen (as of #7024), and tile objects could stick with the clicking on material crafting?

**Comment by PetMudstone** [comment]
Yeah, this issue is mainly relevant to metal sheets and other items that have quick crafting/construction menus upon using them. This is why the name of the issue is so awkwardly worded.

**Comment by corp-0** [comment]
couldn't we put everything in the new crafting screen? We have categories

**Comment by ThatDan123** [comment]
> couldn't we put everything in the new crafting screen? We have categories

Yeah, we should but for new players would be nice if when they are holding a material like metal then the crafting button would flash

And should also open the new crafting menu when clicked?

**Comment by PetMudstone** [comment]
> 
> 
> couldn't we put everything in the new crafting screen?

The new crafting menu is more complex than necessary in regards to using in-hand construction recipes for items like glass or metal. The in-hand construction window for any item are much smaller and simpler than the entire crafting menu.
 If the recipe for table frames were in the new crafting menu then every time you opened it you would have to navigate back to the table recipe, which would presumably sit amongst many other recipes for structures. With the current construction menu that metal rods currently bring up on use have the table recipe is immediately available since only recipes that use metal rods and ONLY metal rods show up.
